dots/.gitconfig

```diff
@@ -0,0 +1,13 @@
+[user]
+  email = mike@npmjs.com
```

Comment:
nit: might want to configure this on a per-repo basis, rather than globally as this is a job specific config